### ISSUE TITLE
[Designer-> Element Properties]: Incorrect name is defined for 'Remove' button under 'Element Properties' pane.[Rendering] Bug Title Here

### ISSUE THREAD

**Issue by Shireen2000** [microsoft/AdaptiveCards#7698]
### Target Platforms

Other

### SDK Version

105.0.1300.0 

### Application Name

Adaptive Cards

### Problem Description

Bug ID: https://microsoft.visualstudio.com/OS/_workitems/edit/33898617

**Test Environment:**
OS Version: Dev (OS build 21390.1)
Browser Version: 93.0.916.1 (Official build) dev(64-bit)
URL: https://adaptivecards.io/designer/
Screen Reader: Narrator, NVDA

**Repro Steps:**
1. Launch the URL: [Adaptive Cards (adaptivecards-ci.azurewebsites.net)](http://adaptivecards-ci.azurewebsites.net/)
2. Navigate to header control ‘Designer’ using tab key and activate it through enter key.
3. Navigate to the "Element properties" button using the TAB key & activate it.
4. Open narrator for the ease of access.
5. Navigate to 'Cross(Remove)' button with Tab key.
6. Observe the screen reader narration.

**Actual Result:**
Incorrect name is defined for 'Remove' button under 'Element Properties' pane. Screen reader is announcing, "Cap X button remove".

**Expected Result:**
Correct name should be defined as 'Remove button' or 'Cancel' for the 'Cross' control. Screen reader should announce as, "Remove/ Cancel button".

**Observation:** Issue is also repro with NVDA (2020.4).

**User Experience:**
Users who rely on screen readers with impairments will effects to navigate if incorrect Name is defined for "Remove" button.

MAS Reference:

https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.01.02

### Screenshots

![MAS4 1 2 - Incorrect name is defined for remove button](https://user-images.githubusercontent.com/96283152/179937782-a2f9617e-3322-4271-862c-db88aaaaa457.png)

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Closing because this is a duplicate of https://github.com/microsoft/AdaptiveCards/issues/7693.

There is also a PR in review where we are discussing the proper resolution to this issue (https://github.com/microsoft/AdaptiveCards/pull/7725)
